package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tokium.co/tokiumsdk",
-  "version": "1.0.15",
+  "version": "1.2.2",
```

Comment:
did we not publish 1.2.3 as the latest version of the sdk?

Reply:
I think something is wrong with it though maybe, gotta make sure. It's fine for now